autocertkit/utils.py

```diff
@@ -1792,6 +1809,47 @@ def deploy_two_droid_vms(session, network_refs, sms=None):
         # their own interfaces (avoid interface forwarding).
         call_ack_plugin(session, 'reset_arp', {'vm_ref': vm_ref})
 
+def deploy_two_droid_vms(session, network_refs, sms=None):
+    """A utility method for setting up two VMs, one on the primary host, and one on a slave host"""
+
+    host_master_ref, host_slave_ref, vm1_ref, vm2_ref = import_two_droid_vms(session, network_refs, sms)
+    config_networks_for_droid_vm(session, vm1_ref, network_refs, 0, sms)
+    config_networks_for_droid_vm(session, vm2_ref, network_refs, 0, sms)
+    start_two_droid_vms(session, host_master_ref, host_slave_ref, vm1_ref, vm2_ref)
+
+    return vm1_ref, vm2_ref
+
+def deploy_two_droid_vms_for_sriov_test(session, vf_driver, network_refs, sms=None):
+    """A utility method for setting up two VMs, one on the primary host for SR-IOV test network,
+    and one on a slave host"""
+
+    host_master_ref, host_slave_ref, vm1_ref, vm2_ref = import_two_droid_vms(session, network_refs, sms)
+
+    # config management network
+    config_network_for_droid_vm(session, vm1_ref, network_refs[1][0], 0, sms)
```

Comment:
Feels confusing as to what `network_refs[1][0]` is now referring to?

Reply:
Originally network_refs is a list contains only two elements -- network references of eth0 and eth1, (master and slave are same)
but for SR-IOV test, this situation changed, network of eth1 for sr-iov exists on only master, so network_refs should contain two list [[network references of eth0, network references of eth1], [network references of eth0, network references of eth1 for sriov]], where network_refs[0] is same/compatible to before,   network_refs[1] is used for create vif/assign vf on master